lib/src/backend/invoker.dart

```diff
@@ -348,6 +352,14 @@ class Invoker {
         await _outstandingCallbacks.noOutstandingCallbacks;
         if (_timeoutTimer != null) _timeoutTimer.cancel();
 
+        if (liveTest.state.result != Result.success &&
+            _runCount < liveTest.test.metadata.retry + 1) {
+          _controller
+              .message(new Message.retry("Retry: ${liveTest.test.name}"));
```

Comment:
My thought process is that we should print the retries in yellow as well. Most teams would likely want to reduce flakiness as much as possible, so having them stand out would be beneficial.